documentation/prerequisites.md

```diff
@@ -7,13 +7,15 @@ Following is the list of prerequisites for running Kubernetes with vSphere Cloud
 * We recommend Kubernetes version 1.6.5 and vSphere version 6.0.x
 * We support Photon, Ubuntu, CoreOs and Centos. it has been tested on Photon 1.0 GA, Ubuntu 16.04, CoreOS 4.11.2 and Centos 7.3
 * vSphere setup to deploy the Kubernetes cluster.
+   - For kubernetes version 1.9.x and above: vSphere Cloud Provider supports kubernetes cluster spnning across multiple vCenters.
+   - For kubernetes version 1.8.x and below: vSphere Cloud Provider supports kubernetes cluster deployed only in one vCenter.
 * vSphere shared storage. It can be any one of VSAN, sharedVMFS, sharedNFS.
```

Comment:
Not your change but can we please reword the following -
vSphere supported storage - It can be HCI offering such as VMware vSAN or block and file storage offerings from storage vendors.

Reply:
I can add HCI offerring part. But I think we should mention VMFS and NFS explicitly. It was very clear while reading.
With new statement we are giving vauge statement .. user will have questions like .. which vendors, which block and file storage offerings? If VMware supports VMFS and NFS (VMware solutions)?

Reply:
Ok